tensorflow/core/kernels/mkl_conv_grad_bias_ops.cc

```diff
@@ -0,0 +1,254 @@
+/* Copyright 2015 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+// See docs in ../ops/nn_ops.cc.
```

Comment:
I realize there is a lot of documentation in nn_ops.cc to which this comment refers. However, could you add comments here that explains how this OpKernel differs from those in 'nn_ops.cc' (it uses mkl library, it changes layout, etc). It would be good to have a high-level description at the top on this file (as well as backprop input).